website/source/docs/providers/aws/d/db_snapshot.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_snapshot"
+sidebar_current: "docs-aws-datasource-db-snapshot"
+description: |-
+  Get information on a DB Snapshot.
+---
+
+# aws\_db\_snapshot
```

Comment:
After reading the code (I read the docs first) this is pretty clearly a singular thing, but the docs for `db_instance_identifier` and `db_instance_identifier` confuse me